test/config-luatest/config_test.lua

```diff
@@ -687,3 +687,81 @@ g.test_security_options = function()
         t.assert_equals(box.cfg.password_history_length, 3)
     end)
 end
+
+g.test_metrics_options_default = function()
+    local dir = treegen.prepare_directory(g, {}, {})
+    local config = [[
+        credentials:
+          users:
+            guest:
+              roles:
+              - super
+
+        iproto:
+          listen: unix/:./{{ instance_name }}.iproto
+
+        groups:
+          group-001:
+            replicasets:
+              replicaset-001:
+                instances:
+                  instance-001: {}
+    ]]
+
+    -- Test defaults.
+    local config_file = treegen.write_script(dir, 'base_config.yaml', config)
+    local opts = {
+        config_file = config_file,
+        alias = 'instance-001',
+        chdir = dir,
+    }
+    g.server = server:new(opts)
+    g.server:start()
+    g.server:exec(function()
+        t.assert_equals(box.cfg.metrics.include, {'all'})
+        t.assert_equals(box.cfg.metrics.exclude, { })
+        t.assert_equals(box.cfg.metrics.labels, {alias = 'instance-001'})
+    end)
+    g.server:stop()
```

Comment:
It looks like [this](https://github.com/tarantool/tarantool/pull/8943/commits/1fe36b80f565db5b76cc42da632bfa1bf60b6a0f#diff-29dffb3fd90865518abc81db0984c1c101cf7be1fa03e45668002c9453ba087cR19-R23) does the work.

Reply:
Oh, right. Its from the previous iteration of the test. Removed `:stop()`.